lang/nl/admin/databases.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+return [
+    'title' => 'Database Hosts',
+    'header' => [
+        'title' => 'Database Hosts',
+        'subtitle' => 'Database hosts waar servers databasen op kunnen maken.',
+    ],
+    'content' => [
+    ],
```

Comment:
This content array is empty

Reply:
Yes that is true, that one needs to be added, but this is from the Ptero /admin view.
When I saw the new filament branch, I stopped working on the /admin view. but this should stay